### ISSUE TITLE
Running uitest with Android 11 APK fails with 'APK is not signed'

### ISSUE THREAD

**Issue by OlaOsttveit** [microsoft/appcenter-cli#1670]
### **Description**
We are trying to run an uitest for an android APK via the appcenter cli, which fails with an 'APK is not signed' error message

### **Repro Steps**
Please list the steps used to reproduce your issue.
/usr/local/bin/appcenter test run uitest --app 'AppName' --devices ' --app-path --test-series 'master' --locale 'en_US' --test-parameter --build-dir --uitest-tools-dir 

### **Expected behavior**
The test run should start

### **Details**
1. Is there a particular beacon/command that you are having trouble with?
    - 'appcenter test run uitest'
2. Which version of the App Center CLI are you using?
    - 2.10.0
3. Which OS version did you experience the issue on?
    - macOS BigSur 11.6
4. What command line/shell are you using?
    - zsh 5.8 (x86_64-apple-darwin20.0)
5. What version of Node.js and NPM/Yarn are you using?
    - node v16.0.0, npm 7.10.0
6. Output of the command:
```
Preparing tests... done.
Validating arguments... done.
Creating new test run... done.
Validating application file... done.
Uploading files... done.
Starting test run... failed.
Error: apk must be signed, install on device will fail otherwise - apksigner returned: DOES NOT VERIFY
ERROR: Target SDK version 30 requires a minimum of signature scheme v2; the APK is not signed with this or a later signature scheme

Further error details: For help, please send both the reported error above and the following environment information to us by going to https://appcenter.ms/apps and opening a support ticket (using the help menu in the upper right corner of the screen)
Environment: darwin
App Upload Id:
Timestamp:
Operation: RunUITestsCommand
Exit Code: 3
User Email:
User Name:
User Id:
```

The given APK is correctly signed, which we have verified by running "apksigner verify -v --min-sdk-version 26 AppName.apk", which outputs:

```
Verifies
Verified using v1 scheme (JAR signing): false
Verified using v2 scheme (APK Signature Scheme v2): true
Verified using v3 scheme (APK Signature Scheme v3): true
Verified using v4 scheme (APK Signature Scheme v4): false
Verified for SourceStamp: false
Number of signers: 1
```

The uitest run worked before but started failing when we started building the APK with target framework Android 11.

**Comment by AnatolyPristensky** [comment]
Hello @OlaOsttveit, thank you for getting in touch with us.
We will check your issue with appropriate team and update this conversation.

**Comment by AnatolyPristensky** [comment]
@OlaOsttveit, could you please share information on what version of Xamarin.UITest you are using?
This issue was fixed in 3.0.14 (https://www.nuget.org/packages/Xamarin.UITest/3.0.14) but it's recommended to update to the latest stable version, or at least 3.0.15.
Please, let us know if these steps didn't help.

**Comment by OlaOsttveit** [comment]
We were using Xamarin.UITest 3.0.7, I upgraded to 3.2.2 and now our tests are running again!
